internal/config/view.go

```diff
@@ -0,0 +1,290 @@
+
+var ViewCmd = &command.Command{
+	Cmd: &cobra.Command{
+		Use:     "view",
+		Short:   "View a list of resource entities in the configuration / View the properties of a particular entity",
+		Example: "flow config view account \nflow config view account <accountname>\nflow config view contract \nflow config view contract <contractname>\nflow config view deployment \nflow config view deployment <networkname>\nflow config view emulator \nflow config view emulator <emulatorname>\nflow config view network \nflow config view network <networkname>",
+		Args:    cobra.MaximumNArgs(2),
+	},
+	Flags: &viewResourceFlags,
+	RunS:  viewResource,
+}
+
+func viewResource(args []string,
+	_ flowkit.ReaderWriter,
+	globalFlags command.GlobalFlags,
+	_ *services.Services,
+	state *flowkit.State,
+) (command.Result, error) {
+
+	// If the user does not input any argument, then we print a message with usage example
+	if len(args) == 0 {
+		return &Result{
+			result: "Please enter resource or resource + field arguments \nUsage examples: \n1) flow config view account --> shows the list of all accounts\n2) flow config view account <accountname> --> shows the properties of <accountname>",
+		}, nil
+	}
+
+	// If the argument passed is "account".
+	if args[0] == "account" {
+
+		// Count variable for printing format.
+		var count int = 0
+
+		// IF CONDITION:
+		// If there is one argument in the command i.e. command looks like --> flow config view account,
+		// Then we print the list of all the present accounts in the configuration.
+		// ELSE IF CONDITION:
+		// If there are arguments == 2 i.e. command looks like --> flow config view account <accountname>,
+		// Then we print all the details of the account "<accountname>".
+		// 	If the <accountname> doesn't exist in the configuration, then we print "Account <accountname> does not exist".
+
+		// Declaring a string variable to print our final list.
+		var list string = ""
+
+		if len(args) == 1 {
+			for _, value := range *state.Accounts() {
+				if count == 0 {
+					list = list + string(value.Name())
+					count = count + 1
+				} else if count > 0 {
+					list = list + ", " + string(value.Name())
+				}
+			}
+			// Return the list of accounts.
+			return &Result{
+				result: fmt.Sprintf("List of Accounts: %s", list),
+			}, nil
+		} else if len(args) == 2 {
+			for _, value := range state.Config().Accounts {
+				if value.Name == args[1] {
+					// Return the properties of a particular account.
+					return &Result{
+						result: fmt.Sprintf("Account Name: %s \nAddress: %s \nKey Properties: \nType: %s, Index: %d, Signature Algorithm: %s, Hash Algorithm: %s, Private Key: %s", value.Name, value.Address, value.Key.Type, value.Key.Index, value.Key.SigAlgo, value.Key.HashAlgo, value.Key.PrivateKey),
+					}, nil
+				}
+			}
+			return &Result{
+				result: fmt.Sprintf("Account %s does not exist", args[1]),
+			}, nil
+		}
+	}
+
+	// If the argument passed is "network".
+	if args[0] == "network" {
```

Comment:
It seems that each of these blocks does the same thing with different values which is a good idea to extract into a function.